be/src/olap/in_list_predicate.cpp

```diff
@@ -155,18 +156,18 @@ IN_LIST_PRED_COLUMN_BLOCK_EVALUATE(NotInListPredicate, ==)
                     new_size += _opposite ? !ret : ret;                                            \
                 }                                                                                  \
             }                                                                                      \
-            *size = new_size;                                                                      \
```

Comment:
why delete the code ?

Reply:
`*size = new_size;`
have after `if else`, not needed here

Reply:
ok